recipes/maturin/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -ex
+
+if [ `uname` == Darwin ]; then
+  export MACOSX_DEPLOYMENT_TARGET=10.13
```

Comment:
Is this needed?

Reply:
It's not. I'll remove it in the next commit